README.md

```diff
@@ -42,10 +42,12 @@ To run the app from source, just clone the code and run this in your terminal:
 
 ```
 npm install
-npm run build
```

Comment:
Yeah, I knew it happened before publication time (hence the name). I didn't realise it ran on install as well. Interesting.